distributed/scheduler.py

```diff
@@ -1842,6 +1843,10 @@ def __init__(
                 + repr(self.WORKER_SATURATION)
             )
 
+    @abstractmethod
+    def log_event(self, topic: str | Collection[str], msg: Any) -> None:
```

Comment:
I don't like that this is necessary, but I wouldn't want to do anything about this _in this PR_.